OsmAnd/src/net/osmand/aidl/OsmandAidlApi.java

```diff
@@ -1967,6 +1966,47 @@ public boolean processResult(GPXFile result) {
 		gpxAsyncLoaderTask.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
 	}
 
+	private static int partsCounter = 0;
+	private static long startTime = 0;
+
+	boolean appendDataToFile(final String filename, final byte[] data) {
+		File file = app.getAppPath(IndexConstants.TILES_INDEX_DIR + filename);
```

Comment:
?? (IndexConstants.TILES_INDEX_DIR ?? - not the root file? We don't want to copy other files as well?